scripts/download-os-images.sh

```diff
@@ -35,8 +35,8 @@ if is_linux; then
     "${OS_IMAGE_CHECKSUM_FILE_PATH}"
 elif is_macos; then
   shasum \
-    --algorithm "256" \
```

Comment:
No need to change these. MacOS `shasum` supports these.

Reply:
reverted